ci-operator/step-registry/multiarch/tuning-operator/deploy/multiarch-tuning-operator-deploy-ref.yaml

```diff
@@ -0,0 +1,19 @@
+ref:
+  as: multiarch-tuning-operator-deploy
+  from_image:
+    namespace: origin
+    name: operator-sdk-mto
+    tag: main
+  cli: latest
+  commands: multiarch-tuning-operator-deploy-commands.sh
+  grace_period: 10m
+  resources:
+    requests:
+      cpu: 100m
+      memory: 200Mi
+  env:
+  - name: MTO_OPERATOR_SOURCE
+    default: "latest"
+    documentation: "the value of MTO_OPERATOR_SOURCE is latest or konflux, default is latest"
```

Comment:
What if set this to be:
- bundle: uses a bundle and operator-sdk run bundle
- catalog

And then add:
- KUSTOMIZE_ENV: one of the envs in https://github.com/openshift/multiarch-tuning-operator/tree/main/deploy/envs (prow, konflux)
- CATALOG_IMAGE_OVERRIDE: defaults to "", if filled we add a patch like below
- BUNDLE_OVERRIDE: defaults to "", if filled, we use this instead of $OO_BUNDLE (i.e., `operator-sdk run ${BUNDLE_OVERRIDE:-${OO_BUNDLE}}`)

If CATALOG_IMAGE_OVERRIDE is not empty, we add this patch to the kustomization:

```
patches:
  - target:
      group: operators.coreos.com
      version: v1alpha1
      kind: CatalogSource
      name: multiarch-tuning-operator-catalog
    patch: |-
      - op: replace
        path: /spec/image
        value: ${CATALOG_IMAGE_OVERRIDE}
```

we can then create a new env in the repo, konflux-bundle, that doesn't deploy the CatalogSource, Subscription and OperatorGroups but deploys namespace and imagedigestmirrorset and use `BUNDLE_OVERRIDE` to trigger a test with a bundle, for example, produced by konflux.